app/io/pathfinder/routing/Action.scala

```diff
@@ -0,0 +1,58 @@
+package io.pathfinder.routing
+
+import io.pathfinder.models.Commodity
+import play.api.libs.json.{Writes, Json, JsPath, Format}
+
+sealed abstract class Action(val name: String) {
+    def latitude: Double
+    def longitude: Double
+    def commodity: Long
+}
+
+object Action {
```

Comment:
What would the timestamp represent? Would it be the time we expect the action to occur?